### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by mmcky**
Update the documentation to include

- [x] inequality
- [x] nelder mead (source)
- [x] repeated_game (source)

### CODE REVIEW

**Comment by rht** [comment]
@mmcky https://travis-ci.org/QuantEcon/QuantEcon.py/jobs/466292136 is the test flake I encountered. It is indeed non-deterministic? The nelder mead code is recent, isn't it?

File: quantecon/inequality.py [hunk 1]
```diff
@@ -106,10 +106,11 @@ def shorrocks_index(A):
         An index equal to 0 indicates complete immobility.
```
Comment (mmcky) [inline quantecon/inequality.py]: @natashawatkins could you check this math equation. It seems to be breaking on the `html` build.

**Comment by mmcky** [comment]
the build of this branch is viewable: https://quanteconpy.readthedocs.io/en/update-docs/

**Comment by mmcky** [comment]
thanks @rht. @QBatista can you take a look at the issue with `nelder_mead` test?

File: quantecon/inequality.py (hunk 1 above)
Comment (mmcky) [reply quantecon/inequality.py]: https://quanteconpy.readthedocs.io/en/update-docs/tools/inequality.html

**Comment by mmcky** [comment]
@natashawatkins I tried a few things re: `sharrock` but there are still issues with the build. 

```bash
/home/mmcky/anaconda3/lib/python3.6/site-packages/quantecon-0.4.2-py3.6.egg/quantecon/inequality.py:docstring of quantecon.inequality.shorrocks_index:34: WARNING: Error in "rubric" directive:
no content permitted.

.. rubric:: References

    .. [R23] Benhabib, Bison, and Luo, Wealth distribution and social mobility 
           in the US:  A quantitative approach (2017), https://www.econ.nyu.edu/user/bisina/RevisionAugust.pdf
```
If you can take a look at the RST that would be great.

File: quantecon/inequality.py (hunk 1 above)
Comment (natashawatkins) [reply quantecon/inequality.py]: How can I actually build the docs myself because I was having problems when I followed the instructions?

**Comment by natashawatkins** [comment]
I think the above should have fixed it, but I can't test because docs won't compile

I'm not sure about the math error - it renders perfectly fine in latex

**Comment by natashawatkins** [comment]
I'll try double escaping `\f` since it seems to have a problem with that

**Comment by natashawatkins** [comment]
Actually I just added a raw flag

**Pull request merged**